heutagogy/views.py

```diff
@@ -50,11 +50,13 @@ class Bookmarks(Resource):
     @token_required
     def get(self):
         url = request.args.get('url')
+        tags = request.args.getlist('tag')
 
-        filters = dict(user=current_user.id)
+        filters = [db.Bookmark.user == current_user.id]
```

Comment:
I forgot the python syntax :see_no_evil: 

Is `[ True | False ]` the possible values after executing `[db.Bookmark.user == current_user.id]`?

Reply:
Column types override `==` operator to return an object (not bool).

Reply:
Das ist interessant.